### ISSUE TITLE
How to replace chained method call

### ISSUE THREAD

**Issue by othercorey** [rectorphp/rector-cakephp#7]
@TomasVotruba 

We have an API change in CakePHP 4.3:

From:
```php
$this->getTableLocator()->get('Table');
```
To:
```php
$this->fetchTable('Table');
```

Is there a rector rule that allows this?

This is available for a specific Trait, but we do have classes we know use the trait if that's not possible.

**Comment by TomasVotruba** [comment]
Hi, thanks for great question 👍

First you need to find method call, where:

- `name` is "get"
- `var` is an instance of `MethodCall`

Then return a `new MethodCall()` node and see what happens. The rest will be clear to you ☺️

**Comment by markstory** [comment]
This can be fixed with #9
